modal/app.py

```diff
@@ -616,7 +616,7 @@ def function(
         schedule: Optional[Schedule] = None,  # An optional Modal Schedule for the function
         secrets: Sequence[_Secret] = (),  # Optional Modal Secret objects with environment variables for the container
         gpu: Union[
-            GPU_T, Sequence[GPU_T]
+            GPU_T, List[GPU_T]
```

Comment:
I think this is better as we don't mean for the `Sequence` to be a tuple or a string or anything else that implements the sequence interface. We just want a plain `list`.

Reply:
Maybe `region: Optional[Union[str, Sequence[str]]] = None,` should be made as a `List` too - was doing it to stay internally consistent.